src/utils/validateInputWithRegex.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { validate, validateAndCleanRegexInput } from "./validateInputWithRegex";
+import { EnvVersionError } from "../services/envVersion";
+
+describe("validate", () => {
+  const alphanumericRegex = "^[a-zA-Z0-9-_ ]+$";
+  it("should return true for alphanumeric regex", () => {
+    expect.assertions(4);
```

Comment:
Ooh interesting! What does this do haha

Reply:
this is used to ensure a certain number of assertions are called in a test case!😀

Reply:
oooh, got it! haha til xD